app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java

```diff
@@ -71,32 +72,33 @@ public DeleteHelper(MediaWikiApi mwApi,
      * @param reason
      * @return
      */
-    private boolean delete(Media media, String reason) {
+    private boolean delete(Context context, Media media, String reason) {
         String editToken;
         String authCookie;
-        String summary = "Nominating " + media.getFilename() + " for deletion.";
+        String summary = context.getString(R.string.delete_helper_delete_summary_1)
+                + media.getFilename() + context.getString(R.string.delete_helper_delete_summary_2);
 
         authCookie = sessionManager.getAuthCookie();
         mwApi.setAuthCookie(authCookie);
 
         Calendar calendar = Calendar.getInstance();
-        String fileDeleteString = "{{delete|reason=" + reason +
```

Comment:
Hi  @PavelAplevich , I don't think this should go to strings.xml. In string xml there shouldn't be any code (or markup language) like content. Strings are what translators will see and they should include just natural language expressions. So please revert any changes which does not fit this description.